<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1107</width>
    <height>910</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="showText">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>360</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Calculate</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableTest">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>971</width>
     <height>281</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="testButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>360</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Buy Station ID</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="waitBox">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>360</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;wait box&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>146</x>
     <y>10</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Sell Station ID</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Buy Region ID</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>10</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Sell Region ID</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>410</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Broker Fee</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>450</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>transaction_tax</string>
   </property>
  </widget>
  <widget class="QLabel" name="buyFromBuyOrders">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>10</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>buy_from_buy_orders</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>10</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>sell_to_buy_orders</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>10</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>sell_to_region</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>430</y>
     <width>151</width>
     <height>261</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>minimum sell price:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_15">
      <property name="text">
       <string>history_size</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>min daily profit</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>min ISK volume</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>min per day sold</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_16">
      <property name="text">
       <string>days_not_sold_per_month</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPlainTextEdit" name="minSellPrice">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>430</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="plainText">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="minDaySold">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>620</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="plainText">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="minISKVolume">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>560</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="plainText">
    <string>3000000</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="minDailyProfit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>520</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="plainText">
    <string>1000000</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="historySize">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>480</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="plainText">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="daysNotSold">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>660</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="plainText">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="buyStationID">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="plainText">
    <string>60003760</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="sellStationID">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>30</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="plainText">
    <string>1027847407700</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="buyRegionID">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>30</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="plainText">
    <string>10000002</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="sellRegionID">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>30</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="plainText">
    <string>10000014</string>
   </property>
  </widget>
  <widget class="QComboBox" name="buyBuyOrders">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>40</y>
     <width>129</width>
     <height>19</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>False</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>True</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="sellBuyOrders">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>40</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>False</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>True</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="sellToRegion">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>40</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>False</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>True</string>
    </property>
   </item>
  </widget>
  <widget class="QPlainTextEdit" name="brokerFee">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>410</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="plainText">
    <string>0.03</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="transactionTax">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>450</y>
     <width>99</width>
     <height>31</height>
    </rect>
   </property>
   <property name="plainText">
    <string>0.014</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="redirect">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>530</y>
     <width>104</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="clientID">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>620</y>
     <width>104</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="secretKey">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>710</y>
     <width>104</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="refreshToken">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>800</y>
     <width>104</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>650</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Client ID</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>256</x>
     <y>732</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>secret key</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>256</x>
     <y>543</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>redirect uri</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>236</x>
     <y>822</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>refresh token</string>
   </property>
  </widget>
  <widget class="QPushButton" name="updateToken">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>620</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Update Refresh Token</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loginButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>460</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Login(required to get refresh token)</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="loginURL">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>490</y>
     <width>651</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="refreshURL">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>570</y>
     <width>641</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>543</y>
     <width>401</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Put the URL you ended up after logging in in here then click update refresh token</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>showText</tabstop>
  <tabstop>testButton</tabstop>
  <tabstop>waitBox</tabstop>
  <tabstop>tableTest</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
